CSharp/README.md

```diff
@@ -37,7 +37,8 @@ This sample demonstrates efficient list queries and provides a utility library f
 This set of samples is intended to be the starting point for learning the concepts behind Azure Batch and its API. It covers basic features of the service, including jobs, pools, tasks, and more.
 
 ### [TextSearch](./TextSearch)
-This map-reduce style sample uses Azure Batch to perform parallel text processing on an input file by splitting it up into multiple sub-files and performing regular expression matching on each sub-file. A Job Manager task is used to orchestrate the mapper and reducer tasks with the results rolled-up into a final report. See the readme in the source directory for more information.
+This map-reduce style sample uses Azure Batch to perform parallel text processing on an input file by splitting it up into multiple sub-files and performing regular expression matching on each sub-file.
+This sample makes use of a variety of Azure Batch features, such as auto-pool functionality, task `ResourceFiles` for moving data into VMs, task `OutputFiles` for moving data out of VMs, and task dependencies for specifying tasks relationships to one another. See the readme in the source directory for more information.
```

Comment:
is readme technically supposed to be lowercased or upper? I honestly don't know.

Reply:
Like all upper? README?

I'm pretty sure we're not consistent, but I'll go fix casing to be the same as the actual file on disk (whatever case that is).

Reply:
Oh, it's lowercase on disk there, so at least we're matching... I think I'll leave this as is?

Reply:
Yea, just kinda curious. As long as we are consistent its fine